### ISSUE TITLE
2.1.1. doesn't accurately summarise the WCAG success criterion

### ISSUE THREAD

**Issue by jfhector** [alphagov/wcag-primer#35]
The issue is both on the index page and on the details page for 2.1.1.

I have submitted a pull request to fix it: https://github.com/alphagov/wcag-primer/pull/34

###### Current phrasing

"Make sure every task can be completed without a mouse."

###### Problem

This doesn't work as an accurate summary of the success criterion, because:
* it doesn't mention keyboard at all, which is what this requirement is about
* it could suggest that if every task can be completed with a touch screen, then the criterion would be met. This is not the case.

###### Proposed solution

I propose rephrasing the summary to:
"Make sure every task can be completed using just a keyboard."

###### Current phrasing

"It must be possible for someone using a keyboard or touch device to complete all tasks in a service. This ensures that people with mobility impairments who do not use a mouse can successfully complete their goals."

"Requirements / What to do?
* All interaction and functionality is usable with a keyboard;
* All interaction and functionality is usable on a touch-screen device."

###### Problem

This is confusing in the same way. It suggests that if a user can complete tasks using a touch screen device, the success criterion is met. But in the official W3C WCAG, this success criterion is not about touch screen at all.

###### Proposed solution

I suggest rephrasing the intro paragraph to this:

"It must be possible for someone using only a keyboard (or a device that emulates keyboard commands) to complete all tasks in a service. This ensures that people with mobility or dexterity impairments who do not use a mouse can successfully complete their goals."

I also suggest deleting the second requirement about touch-screen device.

**Comment by jfhector** [comment]
Linking in @mfairchild365 as we've reviewed this together.

@BNewing @svinkle @patrickhlauke @36degrees do you see any problem with these changes? I'm not attached to what wording we use, but I'd love to fix the issue.

**Comment by patrickhlauke** [comment]
i agree with the proposed change. it reflects the actual SC, where the current wording doesn't.

**Comment by svinkle** [comment]
Looks good. 👍

**Comment by BNewing** [comment]
Looks good to me too :)

**Comment by chris-gds** [comment]
✅ Replicated: https://github.com/alphagov/wcag-primer/pull/72 - 
Closing issue ahead of merge.
